model/model_training/configs/config.yaml

```diff
@@ -98,7 +98,7 @@ oasst_export_eu:
     - alpaca
     - oig_file:
         source_url: https://huggingface.co/datasets/laion/OIG/resolve/main/unified_chip2.jsonl
-        max_count: 30000
+        max_count: 10000
         min_length: 1000
         val_split: 0.2
```

Comment:
Same here, val_split 0.1 is probably enough

Reply:
When pre-training works I will probably remove this here .. the GPT4all dataset re-generade all chip answers with GPT3.5 .. we'll use them.